core/gateway.go

```diff
@@ -237,17 +237,25 @@ func wrapSolveError(inputErr *error, gw bkgw.Client) {
 		if err != nil {
 			return
 		}
-		var exitErr *bkpb.ExitError
-		if err := proc.Wait(); !errors.As(err, &exitErr) {
+		if err := proc.Wait(); err != nil {
 			return
 		}
 
+		stdout := strings.TrimSpace(ctrOut.String())
+		stderr := strings.TrimSpace(ctrErr.String())
+
+		exitCode := -1
```

Comment:
This default signals that an exit code wasn't found. Is it better to default to 1?